routing_common/routing_common_tests/transit_test.cpp

```diff
@@ -53,18 +53,47 @@ UNIT_TEST(Transit_HeaderSerialization)
   }
 }
 
+UNIT_TEST(Transit_TitleAnchorSerialization)
+{
+  {
+    TitleAnchor anchor(17 /* min zoom */, 0 /* anchor */);
```

Comment:
Can we define an enum for anchor, because this constants looks magic?

Reply:
Why to define? I'll use enum Anchor from drape/drape_global.hpp for these tests.
Done.